firmware.conf

```diff
@@ -353,11 +353,6 @@ version = 1.89
 url = http://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p1573561412/v127869/CP032172.scexe
 file = ilo3_189.bin
 
-[ilo4]
```

Comment:
Why remove it? iLO 3 has the same duplicate entry. This is the current unversioned default for ilo4.

Reply:
Damn, didn't get that.